docs/book/src/developer/logging.md

```diff
@@ -130,23 +130,26 @@ thorny parts of code. Over time, based on feedback from SRE/developers, more log
 
 ## Developing and testing logs
 
-We are continuously improving our [Tilt](tilt.md) setup allowing Cluster API developers to use logs and improve them as part of
-their development process.
- 
-Developers can deploy an entire log observability suite by simply setting the
-`deploy_observability` value in your [tilt-setting.yaml](tilt.md#create-a-tilt-settings-file).
+Our [Tilt](tilt.md) setup offers a batteries-included log suite based on [Promtail](https://grafana.com/docs/loki/latest/clients/promtail/), [Loki](https://grafana.com/docs/loki/latest/fundamentals/overview/) and [Grafana](https://grafana.com/docs/grafana/latest/explore/logs-integration/).
 
-```yaml
-deploy_observability:
-- promtail
-- loki
-- grafana
-```
+We are working to continuously improving this experience, allowing Cluster API developers to use logs and improve them as part of their development process.
 
-In order to test structured logging it is required to override some controller flags, and similar approach can be used 
-for increasing log verbosity, e.g.
+For the best experience exploring the logs using Tilt:
+1. Set `--logging-format=json`. 
+2. Set a high log verbosity, e.g. `v=5`.
+3. Enable promtail, loki, and grafana under deploy_observability.
 
+A minimal example of a tilt-settings.yaml file that deploys a ready-to-use logging suite looks like:
 ```yaml
+deploy_observability:
+  - promtail
+  - loki
+  - grafana
+  - prometheus
```

Comment:
Out of curiosity: You only need prometheus in the user wants metrics data correct? Is it also used in anyway for logging?

Reply:
AFAIK: ther is no need for prometheus if I only want to take a look at logs.

Reply:
Yeah - prometheus isn't needed here. Removed now.